### PR TITLE
Add cat field to Role Skill

### LINKED ISSUE #129: Add cat enum field to RoleSkill
We're missing the cat enum field for `RoleSkill`.

For #26.

### PR DESCRIPTION
**Pull request by npendery**
Closes #129

### CODE REVIEW

File: web/models/role_skill.ex [hunk 1]
```diff
@@ -15,8 +17,9 @@ defmodule CodeCorps.RoleSkill do
   """
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:role_id, :skill_id])
+    |> cast(params, [:cat, :role_id, :skill_id])
     |> validate_required([:role_id, :skill_id])
+    |> validate_inclusion(:cat, cats)
```
Comment (joshsmith) [inline web/models/role_skill.ex:22]: I would put the `cast` and validation into an `import_changeset` instead. We don't do imports yet, but that would be the ideal place for this.

I added the same migration in my not-yet-merged branch, so I can remove that when this is merged.

**Comment by joshsmith** [comment]
Looks like a bunch of random commits got thrown here. May need to rebase.

**Comment by npendery** [comment]
@JoshSmith yeah I saw that, working on that now. Some funky stuff happened when I rebased from origin so my local develop branch is off.

**Comment by joshsmith** [comment]
It may be less effort to close and copy/paste raw files, depending on how crazy git history is.

**Comment by npendery** [comment]
Yeah thats what I was thinking too, doing that now

**Pull request closed without merging**